### DISCUSSION TITLE
How far away is Pydantic 2 from being released?

### DISCUSSION THREAD

**Discussion by nh916** [pydantic/pydantic#5875] [Question]
I am thinking of using Pydantic 2 in a big project at work and I am wondering how far away is it from being released and if there will be a lot of issues with me using it in the project?

**Comment by adriangb** [comment]
A beta will be released within the next week (probably), but there are already alpha versions available. I'm not sure what your project looks like so I can't say what sort of issues (if any) you will have, but please give it a try and let us know if you encounter any!
